vcpkg/github-integration.md

```diff
@@ -0,0 +1,94 @@
+
+### About the GitHub dependency graph
+
+The GitHub [dependency graph](https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph) stores the set of dependencies for a repository. Beyond being able to visualize a repository's dependencies, GitHub builds several useful features on top of this data, including [dependency review](https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review) and [Dependabot alerts](https://docs.github.com/code-security/dependabot/dependabot-alerts/about-dependabot-alerts).
+
+### vcpkg integration with the GitHub dependency graph
+
+vcpkg has experimental support for populating the GitHub dependency graph from within a GitHub Actions workflow. To enable this feature, make the following changes in your workflow file:
+
+* Set the `VCPKG_FEATURE_FLAGS` environment variable to include the value `dependencygraph`. 
+* Set the `GITHUB_TOKEN` environment variable to the value `${{ secrets.GITHUB_TOKEN }}`.
+* Give GitHub Actions write access to the contents of your repository by including the following permissions block. This is required to write the dependency graph metadata to your repository. This __will not__ add any commits to your repository or make any other modification to the contents of your repository.
+
+```yaml
+permissions:
+  contents: write
+```
+
+You must enable the GitHub dependency graph in your repository's settings (enabled by default on public repositories). For private repositories, follow the [GitHub documentation to enable the dependency graph](https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/configuring-the-dependency-graph#enabling-and-disabling-the-dependency-graph-for-a-private-repository) in order to satisfy this requirement.
+
+### Known limitations
+
+The following are known limitations in the current implementation. __Even though we know of these, if they affect the value for you, we'd still like to hear from you so that we can prioritize what we need to fix and how we should fix it.__
+
+* The version of vcpkg that is bundled with GitHub Actions runners does not have this feature yet. Ensure you are using a version of vcpkg that supports this feature.
+* Features that depend on the dependency graph, such as Dependabot alerts and Dependabot pull requests, are not yet available. Please let us know if you are interested in those features!
+
+### Example GitHub Actions workflow
+
+> [!NOTE]
+> This example assumes that there is a valid `vcpkg.json` manifest that lists some dependent ports
```

Comment:
If I don't have a vcpkg.json manifest with dependent ports, how do I get one? Or am I just dead in the water at this point if I don't?

Reply:
Adding a link to documentation on manifest mode to guide the user if they don't have a manifest in the next update.